atest/testdata/standard_libraries/collections/list.robot

```diff
@@ -672,6 +672,12 @@ List Should Not Contain Duplicates With Ignore Case
 List Should Contain Value With Ignore Case And Nested List and Dictionary
     [Setup]    Create Lists For Testing Ignore Case
     List Should Contain Value  ${L4}  value=d    ignore_case=${True}
+    
+Lists Should be equal with Ignore Case and Order
+    [Setup]    Create Lists For Testing Ignore Case
+    [Template]    Lists Should Be Equal
+    list1=${L7}    list2=${L8}    ignore_order=${True}    ignore_case=${True}
+    list1=${L9}    list2=${L10}    ignore_order=${True}    ignore_case=${True}
```

Comment:
Two comments mostly related to style.

1. Instead of creating these lists that are used just once in a separate keyword, it would probably be simpler to create them here using inline Python evaluation like `${{['apple', 'Banana', 'cherry']}}`. That syntax is somewhat complicated, but here it would allow seeing compared values directly.

2. There's no need to use e.g. `ignore_order=${True}` when simpler `ignore_order=True` works. 

Both of the above notes apply also to many other tests and changing them as part of this PR isn't a good idea. I think I'll merge this now and we can consider cleaning up all tests separately later.